tests/test_commandline.py

```diff
@@ -215,6 +215,22 @@ def _test_erroneous(self):
                 stderr=subprocess.STDOUT,
             )
 
+        try:
+            # No full URI given
+            subprocess.check_output(
+                AIOCOAP_CLIENT + [self.servernetloc + "/empty"],
+                stderr=subprocess.STDOUT,
+            )
+        except subprocess.CalledProcessError as e:
+            self.assertTrue(
+                "Most URLs in aiocoap need to be given with a scheme"
```

Comment:
```suggestion
                "URL incomplete: Must start with a scheme."
```
This switches from `extra_help()` to the exception description which is hopefully less brittle - and much closer to `with self.assertRaises(IncompleteUrlError):` which would be the accurate test. (But going through subprocess is the right call here)

Reply:
Thanks, testing for both now – this should help make sure that I don't lose the extra_help output to some small bug. Of course it's brittle, but then also easy to adjust.